testapi.pm

```diff
@@ -1413,6 +1413,22 @@ sub freeze_vm {
     query_isotovideo('backend_freeze_vm');
 }
 
+=head2 cont_vm
```

Comment:
as we're dragging this into public API, can we please fix the function name to be english?